tests/analyzer/metadata/resources/sample_npm_project_info.py

```diff
@@ -0,0 +1,71 @@
+from copy import deepcopy
+
+NPM_PACKAGE_INFO = {
```

Comment:
any reason why this would be a Python file and not just a .json?

Reply:
no 🤔